src/Microsoft.AspNet.Mvc.Razor/RazorView.cs

```diff
@@ -42,6 +42,9 @@ public class RazorView : IView
             IsPartial = isPartial;
         }
 
+        /// <inheritdoc />
+        public string Path => RazorPage.Path;
```

Comment:
you're trading off an unnecessary delegate (or two, depending on the IL generation) and its backing field against a single backing field?